concert/coroutines/base.py

```diff
@@ -168,6 +171,7 @@ async def feed_queue(producer, func, *args):
         future = loop.run_in_executor(None, func, pqueue, *args)
         prio = 1
         async for item in producer:
+            item = item.view(np.ndarray)
```

Comment:
This is actually not needed anymore, since the time and number should not make such a big influence on the memory consumption.